<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QMainWindow" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1415</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QGroupBox" name="gbParametres">
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Paramètres</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="hlPhotosSelection">
             <item>
              <widget class="QPushButton" name="pbSelectPhotos">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Sélection du dossier de photos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="laPhotosDir">
               <property name="maximumSize">
                <size>
                 <width>500</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="dzqddzqd">
             <property name="title">
              <string>Liste des photos</string>
             </property>
             <layout class="QHBoxLayout" name="hlPhotosList">
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="lineWidth">
                 <number>0</number>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>552</width>
                   <height>146</height>
                  </rect>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QListWidget" name="lwPhotos">
                    <property name="maximumSize">
                     <size>
                      <width>10000</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbConsignes">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="title">
              <string>Edtieur de la page</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="hlConsignes">
                <item>
                 <layout class="QVBoxLayout" name="vlTextEdit">
                  <item>
                   <layout class="QHBoxLayout" name="hlFont">
                    <item>
                     <widget class="QFontComboBox" name="fcbConsignes">
                      <property name="editable">
                       <bool>false</bool>
                      </property>
                      <property name="currentFont">
                       <font>
                        <family>Calibri</family>
                       </font>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="laTaille">
                      <property name="text">
                       <string>Taille</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="sbSizeTexte">
                      <property name="maximumSize">
                       <size>
                        <width>75</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="minimum">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <number>50</number>
                      </property>
                      <property name="value">
                       <number>15</number>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="laCouleur">
                      <property name="text">
                       <string>Couleur</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pbChoosColor">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="cbBold">
                      <property name="text">
                       <string>Gras</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="cbItalic">
                      <property name="text">
                       <string>Italique</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="hlAlignmentText">
                    <item>
                     <widget class="QLabel" name="laTextAlignment">
                      <property name="text">
                       <string>Aliignement de la consigne :</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rbLeftText">
                      <property name="text">
                       <string>à gauche</string>
                      </property>
                      <attribute name="buttonGroup">
                       <string notr="true">buttonGroup</string>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rbRightText">
                      <property name="text">
                       <string>à droite</string>
                      </property>
                      <attribute name="buttonGroup">
                       <string notr="true">buttonGroup</string>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rbCenterText">
                      <property name="text">
                       <string>centré</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                      <attribute name="buttonGroup">
                       <string notr="true">buttonGroup</string>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_6">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="hlImageAlignment">
                    <item>
                     <widget class="QLabel" name="laPhotoAlignment">
                      <property name="text">
                       <string>Alignement de la photo :</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rbLeftImage">
                      <property name="text">
                       <string>à gauche</string>
                      </property>
                      <attribute name="buttonGroup">
                       <string notr="true">buttonGroup_2</string>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rbRightImage">
                      <property name="text">
                       <string>à droite</string>
                      </property>
                      <attribute name="buttonGroup">
                       <string notr="true">buttonGroup_2</string>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rbCenterImage">
                      <property name="text">
                       <string>centré</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                      <attribute name="buttonGroup">
                       <string notr="true">buttonGroup_2</string>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_7">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPlainTextEdit" name="pteConsigne">
                    <property name="plainText">
                     <string>Taper la consigne ici.</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbFormatPDF">
             <property name="title">
              <string>Style du PDF</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="laNbImagesPage">
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Nombre d'images par page :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="sbNbImagesPage">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>5</number>
                  </property>
                  <property name="value">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="laRatio">
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Ratio taille consigne/image :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dsbRatio">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                  <property name="minimum">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>0.900000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QProgressBar" name="pbLoading">
                  <property name="value">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbPreview">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Preview</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbGeneration">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Génération du pdf</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbImage">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>500</height>
           </size>
          </property>
          <property name="title">
           <string>Affichage</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTabWidget" name="tabDisplay">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tabImage">
              <attribute name="title">
               <string>Affichage photo selectionnée</string>
              </attribute>
              <layout class="QVBoxLayout" name="vlImage">
               <item>
                <layout class="QHBoxLayout" name="hlImageActions">
                 <item>
                  <widget class="QPushButton" name="pbRotationLeft">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Rotation 90° gauche</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pbRotationRight">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Rotation 90° droite</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabPreview">
              <attribute name="title">
               <string>Pré-affichage du pdf</string>
              </attribute>
              <layout class="QVBoxLayout" name="vlPreview"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1415</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
